dask_cloudprovider/gcp/utils.py

```diff
@@ -12,3 +12,22 @@ def inner(http, *args, **kwargs):
         return googleapiclient.http.HttpRequest(new_http, *args, **kwargs)
 
     return inner
+
+
+def is_gce_env() -> bool:
```

Comment:
I'm not sure about this function name. I don't think it's immediately obvious what it does.

What about `is_inside_gce` instead?

Reply:
Sure - renamed.

Reply:
Thanks!